cpp/CMakePresets.json

```diff
@@ -130,7 +130,8 @@
     },
     {
       "name": "ninja-debug-skyhook",
-      "description": "Builds Skyhook libraries along with arrow",
+      "description": "Builds Skyhook libraries",

```

Comment:
I thought I changed all of these, but this got left over :) Thank you!

Reply:
No problem. :-)
I'll merge this once CI is finished.